lib/spack/spack/util/executable.py

```diff
@@ -82,48 +82,48 @@ def path(self) -> str:
     def __call__(
         self,
         *args: str,
-        fail_on_error: bool = True,
-        ignore_errors: Union[int, Sequence[int]] = (),
-        ignore_quotes: Optional[bool] = None,
-        timeout: Optional[int] = None,
-        env: Optional[Union[Dict[str, str], EnvironmentModifications]] = None,
-        extra_env: Optional[Union[Dict[str, str], EnvironmentModifications]] = None,
-        input: Optional[TextIO] = None,
+        fail_on_error: bool = ...,
+        ignore_errors: Union[int, Sequence[int]] = ...,
+        ignore_quotes: Optional[bool] = ...,
+        timeout: Optional[int] = ...,
+        env: Optional[Union[Dict[str, str], EnvironmentModifications]] = ...,
+        extra_env: Optional[Union[Dict[str, str], EnvironmentModifications]] = ...,
+        input: Optional[TextIO] = ...,
```

Comment:
(minor) I agree on the default values, but shouldn't we remove the two below too?